packages/charts/src/charts/circular-chart/tooltip-item.component.tsx

```diff
@@ -25,10 +25,16 @@ const TooltipItem: FC<Props> = ({ data, theme = defaultTheme }) => {
     <Text {...tooltipSettings.labels.typography}>{data}</Text>
   ) : (
     <Container>
-      <Text {...tooltipSettings.labels.typography}>{data.label}</Text>
+      <Text {...tooltipSettings.labels.typography}>
+        {data.label}
+        <span>:&nbsp;</span>
```

Comment:
Have You tried to use `{' '}` ? This will not require creating additional `<span />` element

Reply:
I did, but I'm using flexbox and this empty space is removed so I switched to non-breaking space

Reply:
👍